gas/testsuite/ChangeLog.COREV

```diff
@@ -1,3 +1,38 @@
+2020-11-19  Jessica Mills  <jessica.mills@embecosm.com>
```

Comment:
You only list the new files, not the files you have changed (e.g. `cv-hwloop-09.s`)

Reply:
Now corrected